web/src/routes/admin/system-settings/+page.svelte

```diff
@@ -72,8 +72,8 @@
     },
     {
       item: LibrarySettings,
-      title: 'Library',
-      subtitle: 'Manage library settings',
+      title: 'External Library',
+      subtitle: 'Manage external library settings',
       key: 'library',
```

Comment:
might as well change this to `external-library` as well.

Reply:
Updated 👍